include/EffectView.h

```diff
@@ -29,12 +29,16 @@
 #include "AutomatableModel.h"
 #include "PluginView.h"
 #include "Effect.h"
+#include "EffectLabelButton.h"
```

Comment:
If the friend statement is removed this can be forward declared as well.

Reply:
what would this look like? I tried removing this and adding a `class EffectLabelButton;` line but got a bunch of errors about incomplete types.

Reply:
Did you move the include into file where the member is used, i.e. in the cpp file I assume?

Reply:
Fixed with commit e6aa0ea051035fa72217ed84c2d862b2618f6e86.